boards/arm64/rcar_salvator_xs_m3/salvator_xs_m3-pinctrl.dtsi

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77961.h>
+#include <zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h>
```

Comment:
As said in previous comment, I know it's creating code duplication, but I prefer this board to keep only references to "r8a77961" soc.
Mixing references to both "r8a77951" & "r8a77961" may confused anyone reading board description.

So please, as in Linux, keep `include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77961.h` and the reference to it here.

(ref. https://elixir.bootlin.com/linux/latest/source/drivers/pinctrl/renesas/pfc-r8a7796.c)

Reply:
done